packages/ember-data/lib/adapters/rest-adapter.js

```diff
@@ -391,7 +391,7 @@ export default Adapter.extend(BuildURLMixin, {
     @return {Promise} promise
   */
   findQuery: function(store, type, query) {
-    var url = this.buildURL(type.typeKey, null, null, 'findQuery');
+    var url = this.buildURL(type.typeKey, query, null, 'findQuery');
```

Comment:
ok
I just thought I would check. I don't understand how putting the using `query` instead of `null` would break anything that using `query` instead of `id` would, but it sounds like you've thought about this more in depth than I have.

Anyway, I'm just happy to have the query available in buildURL at all. Thanks!